### PR TITLE
#19402. Added Einsum function to rawops

### LINKED ISSUE #19402: Einsum

### PR DESCRIPTION
**Pull request by janchilling**
Close #19402

### CODE REVIEW

**Comment by theRealBird** [comment]
Hi @janchilling, 
Thank you for your contribution to Ivy, any information about the unsupported data-types which needs to be added here?

**Comment by janchilling** [comment]
> Hi @janchilling, Thank you for your contribution to Ivy, any information about the unsupported data-types which needs to be added here?

Thank you for checking the PR. Since this is a rawops implementation, which refers to the Einsum in general functions so isn't the information about the unsupported datatypes be included in the general functions rather at rawops?

**Comment by janchilling** [comment]
Would appreciate if I can get a reply. Thank you.

**Comment by janchilling** [comment]
@theRealBird Can you please provide an review or request a change please :smile:

**Comment by theRealBird** [comment]
Hi @janchilling, thank you so much for your patience. I really appreciate it.

Explicitly mentioning supported dtypes seems to be the norm for other raw_ops implementations as well, you are right, ideally the information should go in the frontend implementation, but currently that is not present as you might have also seen. I think it would be best to include it here just as a fail-safe.

Thanks!

File: ivy/functional/frontends/tensorflow/raw_ops.py
Comment (theRealBird) [inline ivy/functional/frontends/tensorflow/raw_ops.py:1]: Please aslo resolve the merge conflicts. Thanks!

**Review by theRealBird — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/tensorflow/raw_ops.py
Comment (janchilling) [reply ivy/functional/frontends/tensorflow/raw_ops.py:1]: @theRealBird I resolved the merge conflicts and added the supported data types for the function. Please update me if I have to make anymore changes.

**Comment by janchilling** [comment]
@theRealBird Can I get an update please, Thank you!

**Review by theRealBird — APPROVED** [review]
Thanks @janchilling

**Comment by theRealBird** [comment]
Hi @janchilling,
Thank you for your patience and making the changes!

**Pull request merged**